owm2ffmap/requirements.txt

```
@@ -1,3 +1,6 @@
-tornado
+tornado 5.x 
```

requirements.txt is parsed by pip (during `pip install -r requirements.txt`), so that must conform to pip standards.
```suggestion
tornado~=5.0
```

---

Okay, so I think we should take ```tornado~=5.1.1``` which is the most recent version of that branch. I used it for testing on my machine, so it should work well.

---

`~=` selects the most recent major version compatible with the given version, so that's already done. :)